pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -78,7 +94,7 @@ func (pc *PackageControllerClient) InstallController(ctx context.Context) error
 	return nil
 }
 
-func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Context) error {
+func (pc *PackageControllerClient) IsInstalled(ctx context.Context) error {
 	found, _ := pc.kubectl.GetResource(ctx, "packageBundleController", packagesv1.PackageBundleControllerName, pc.kubeConfig, constants.EksaPackagesName)
 	if found {
```

Comment:
You'll need to change the logic here too given we're now inverting the names semantics.

Reply:
The logic doesn't change actually. The logic is throw an error if the controller is already installed.